products/cfengine.md

```diff
@@ -12,6 +12,11 @@ permalink: /cfengine
 link: https://cfengine.com
 
 releases:
+  - releaseCycle: "3.19"
+    release: 2020-12-10
+    latest: "3.19.0"
+    eol: true
```

Comment:
this `eol: true` means its life ended  so i think you need to make it `eol: false`
and release date looks like wrong , is it released 2 days ago or 1 year ago ? i mean is it released in **2020 or 2021** ?

Reply:
Yes, good spot on the date, 2021-12-10 should the be release date.

However, 3.19.0 is the only release of 3.19.x, it's not supported in that there is no branch where bug fixes are released (like 3.17.0 and 3.16.0) we only release patch updates for LTS series like 3.15.x, 3.18.x.

Reply:
Pushed fix for 3.19.0 release date

Reply:
brother when you write eol: true it marks it already reached to eol  and that means it is not recommend to use